mu-plugins/blocks/screenshot-preview/js/block.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useRef, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import getCardFrameHeight from './get-card-frame-height';
+import useInView from './in-view';
+import ScreenShot from './screenshot';
+
+function Block( { link, previewLink, version } ) {
+	const wrapperRef = useRef();
+	const [ frameHeight, setFrameHeight ] = useState( '1px' );
+	const isVisible = useInView( { element: wrapperRef } );
+	const [ shouldLoad, setShouldLoad ] = useState( false );
+
+	useEffect( () => {
+		const handleOnResize = () => {
+			try {
+				setFrameHeight( getCardFrameHeight( wrapperRef.current.clientWidth ) );
+			} catch ( err ) {}
+		};
+
+		handleOnResize();
+
+		window.addEventListener( 'resize', handleOnResize );
+
+		return () => {
+			window.addEventListener( 'resize', handleOnResize );
```

Comment:
If this is intended to clean up should it be using `removeEventListener`?;

Reply:
Great catch!